ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml

```
@@ -50,6 +50,42 @@ tests:
         source /var/run/secrets/sbo-on-sb-perf/registry-credentials-env.sh
         make -k test-performance test-performance-collect-kpi || true
         make test-performance-artifacts
+        make test-performance-thresholds
+      credentials:
+      - mount_path: /var/run/secrets/sbo-on-sb-perf
+        name: sbo-on-sb-perf
+        namespace: test-credentials
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+      timeout: 5h0m0s
+    workflow: ipi-aws
+  timeout: 8h0m0s
+- as: performance-current-dev-sandbox
+  cron: 0 0 * * 1
```

Q: is this midnight UTC? If so, it may make sense to start these a few hours earlier to account for India coming online.  Something like
```suggestion
  cron: 0 20 * * 0
```

---

@sadlerap Yes, good point... updated accordingly. Thanks!